samigo/samigo-app/src/webapp/jsf/section-activity/sectionActivityHeadings.jsp

```diff
@@ -8,9 +8,9 @@
        <h:outputText value="#{generalMessages.assessment}" />
        <f:actionListener type="org.sakaiproject.tool.assessment.ui.listener.author.AuthorActionListener" />
     </h:commandLink>
-<f:verbatim></span></li>
+<f:verbatim rendered="#{authorization.adminTemplate and template.showAssessmentTypes}"></span></li>
```

Comment:
This change controls this:

<f:verbatim rendered="#{authorization.adminTemplate and template.showAssessmentTypes}"></span></li>
<li role="menuitem" ><span></f:verbatim>

That’s close for the preceding item and open for the next. If you conditionalize the command link but not this, you get a blank <li><span>. That results in too much space. It produces an inconsistency with the other tabs

> On Mar 31, 2016, at 11:37 AM, Sam Ottenhoff <notifications@github.com> wrote:
> 
> In samigo/samigo-app/src/webapp/jsf/section-activity/sectionActivityHeadings.jsp <https://github.com/sakaiproject/sakai/pull/2079#discussion_r58075710>:
> 
> > @@ -8,9 +8,9 @@
> >         <h:outputText value="#{generalMessages.assessment}" />
> >         <f:actionListener type="org.sakaiproject.tool.assessment.ui.listener.author.AuthorActionListener" />
> >      </h:commandLink>
> > -<f:verbatim></span></li>
> > +<f:verbatim rendered="#{authorization.adminTemplate and template.showAssessmentTypes}"></span></li>
> So this change above controls the rendering of an end span tag only ...... This doesn't make any sense to me.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub <https://github.com/sakaiproject/sakai/pull/2079/files/a04e7f41bf3b0921b76db3da76cf541c2dd1e031#r58075710>